### ISSUE TITLE
refreshAllItems() (android)

### ISSUE THREAD

**Issue by KjellConnelly** [dooboolab/react-native-iap#25]
I know that this function shouldn't work all the way when using the emulator, but for testing purposes, it seems it should call the catch function. Not sure if it doesn't work on both emulator and real device, but it doesn't work on emulator it seems.

```javascript
async function restore() {
  try {
    const results = await RNIap.refreshAllItems()
    alert("Got results")
   } catch(error) {
        // this gets called on ios physical devices, but not android emulator
   }
}
```
This function never ends since it is indefinitely waiting for results or an exception.

**Comment by dooboolab** [comment]
@KjellConnelly 
Will [this](https://github.com/dooboolab/react-native-iap/commit/a2e597011b16eca3c2b33ea4c34ab2f35d665630) be a fix?

**Comment by dooboolab** [comment]
@KjellConnelly 
I've just published to `0.2.9`. Please try again with new version.
I'll close this. Tell me again if you still have a problem.
